docusaurus/docs/dev-docs/migration/v4-to-v5/breaking-changes/database-identifiers-shortened.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Database identifiers shortened in v5
+description: Database identifiers are shortened in Strapi v5 and can't be longer than 53 characters to avoid issues with identifiers that are too long.
+sidebar_label: Database identifiers shortened
+displayed_sidebar: devDocsMigrationV5Sidebar
+tags:
+ - breaking changes
+ - database
+---
+
+import Intro from '/docs/snippets/breaking-change-page-intro.md'
+import MigrationIntro from '/docs/snippets/breaking-change-page-migration-intro.md'
+
+# Database identifiers shortened in v5
+
+In Strapi v5, database identifiers can't be longer than 53 characters. <Intro />
+
+## Breaking change description
+
+<SideBySideContainer>
+
+<SideBySideColumn>
+
+**In Strapi v4**
+
+Database identifiers could be longer than 53 characters, potentially causing issues with some databases.
+
+</SideBySideColumn>
+
+<SideBySideColumn>
+
+**In Strapi v5**
+
+Database identifiers can't be longer than 53 characters and will be shortened.
+
+</SideBySideColumn>
+
+</SideBySideContainer>
+
+## Migration
+
+<MigrationIntro />
+
+### Notes
+
+- A hashing key will be added when shortening database identifiers to avoid conflicts. It will consist in the first 6 characters of SHA-256. For example, `my_very_very_very_very_very_very_very_too_long_identifier_unique` will be shortened to `my_very_very_very_very_very_very_very_very_a2d3x3_unq` in Strapi v5.
```

Comment:
```suggestion
- A hashing key will be added when shortening database identifiers to avoid conflicts. It will consist in the first 6 characters of SHA-256. For example, `my_very_very_very_very_very_very_very_too_long_identifier_unique` will be shortened to `my_very_very_very_very_very_very_very_very_06a6ab_unq` in Strapi v5.
```